### ISSUE TITLE
Crash on login to Twitter or TweetDeck

### ISSUE THREAD

**Issue by kris7t** [getferdi/ferdi#1029]
**Bug description:**
When trying to load Twitter in a new service, an empty page is displayed. Trying to change the URL from https://twitter.com to anything else promptly crashes Ferdi with a segmentation fault.

In contrast, TweetDeck loads correctly, but when clicking the Log In with Twitter button, I get redirected to a https://mobile.twitter.com page, which exhibits the same phenomenon as above. However, if I enter a new URL _before_ hitting to Log In, Ferdi can successfully navigate to it, so navigation isn't broken, only navigating away from Twitter.

Curiously, it is possible to open the Service Developer Tools even on the blank page, where everything seems normal (I can reload the page and check network headers, or start profiling the JavaScript being executed). There don't seem to be any unusual errors in the service console, the Ferdi console, or printed to the standard output.

I was trying to track down the crash, but the sheer size of Chromium and the fact I haven't compiled it with debug symbols makes the stack trace impossible to interpret.

```
       Message: Process 27684 (electron) of user 1000 dumped core.

                Stack trace of thread 27684:
                #0  0x000055977bc607ed n/a (electron + 0x345d7ed)
                #1  0x000055977bf3e89e n/a (electron + 0x373b89e)
                #2  0x000055977bf505f2 n/a (electron + 0x374d5f2)
                #3  0x000055977bf573e7 n/a (electron + 0x37543e7)
                #4  0x000055977bc57246 n/a (electron + 0x3454246)
                #5  0x000055977bc56b92 n/a (electron + 0x3453b92)
                #6  0x000055977bc56a2b n/a (electron + 0x3453a2b)
                #7  0x000055977bc24c38 n/a (electron + 0x3421c38)
                #8  0x000055977bc36611 n/a (electron + 0x3433611)
                #9  0x000055977bc35cc6 n/a (electron + 0x3432cc6)
                #10 0x000055977bc36ab5 n/a (electron + 0x3433ab5)
                #11 0x000055977bc5575f n/a (electron + 0x345275f)
                #12 0x000055977a8d68c0 n/a (electron + 0x20d38c0)
                #13 0x000055977c7fe5e8 n/a (electron + 0x3ffb5e8)
                #14 0x000055977c801a7e n/a (electron + 0x3ffea7e)
                #15 0x000055977ca5e6e3 n/a (electron + 0x425b6e3)
                #16 0x000055977ca5c177 n/a (electron + 0x4259177)
                #17 0x000055977c680a39 n/a (electron + 0x3e7da39)
                #18 0x000055977c691b3f n/a (electron + 0x3e8eb3f)
                #19 0x000055977c691893 n/a (electron + 0x3e8e893)
                #20 0x000055977c650787 n/a (electron + 0x3e4d787)
                #21 0x00007f5e9543f924 g_main_context_dispatch (libglib-2.0.so.0 + 0x52924)
                #22 0x00007f5e95493621 n/a (libglib-2.0.so.0 + 0xa6621)
                #23 0x00007f5e9543e131 g_main_context_iteration (libglib-2.0.so.0 + 0x51131)
                #24 0x000055977c6505a1 n/a (electron + 0x3e4d5a1)
                #25 0x000055977c6921c4 n/a (electron + 0x3e8f1c4)
                #26 0x000055977c66a917 n/a (electron + 0x3e67917)
                #27 0x000055977bac06f6 n/a (electron + 0x32bd6f6)
                #28 0x000055977bac2703 n/a (electron + 0x32bf703)
                #29 0x000055977babd768 n/a (electron + 0x32ba768)
                #30 0x000055977b9938ea n/a (electron + 0x31908ea)
                #31 0x000055977b993532 n/a (electron + 0x3190532)
                #32 0x000055977d7fbc7a n/a (electron + 0x4ff8c7a)
                #33 0x000055977acf2784 n/a (electron + 0x24ef784)
                #34 0x000055977a1c0dd4 n/a (electron + 0x19bddd4)
                #35 0x00007f5e91221152 __libc_start_main (libc.so.6 + 0x28152)
                #36 0x000055977a1c0aee _start (electron + 0x19bdaee)
```

**Steps to reproduce:**

1. Add a new Twitter or TweetDeck service.
2. Try to log in.
3. Observe the empty page.
4. If a new URL is entered to the navigation bar, Ferdi crashes.

**Expected behavior**
I can log in to Twitter and Ferdi doesn't crash.

**Environment:**

- Operating System: Arch Linux `Linux 5.9.0-arch1-1 #1 SMP PREEMPT Mon, 12 Oct 2020 18:52:13 +0000 x86_64 GNU/Linux`
- Ferdi Version: 5.6.0-nightly.1 d97c00f962757210d7ef617c40d4c4e3da37efb1
- Electron Version: 9.3.2
- Server: Using without an account

- Debug information: I can't seem to be able to copy or publish debug information, but that might just be a misconfiguration of my environment (to reproducibly confirm the bug, I ran Ferdi directly from the source tree).

**Comment by dbieber** [comment]
I am also observing this behavior: when I click "Log in" w/ Tweetdeck on Mac OS I get a blank page.

**Comment by JPnux** [comment]
I have the same problem with FERDI 5.6.0-beta.5 DEB version on Ubuntu 20.04.1 LTS

**Comment by dbieber** [comment]
From @kris7t on gitter: "the cookies in my profile from a previous version of Ferdi work just fine, so it's not a problem with Twitter, only with the login flow itself."

So there's probably a way to work around the issue by logging in in one browser and then transferring the cookies to Ferdi.

**Comment by JPnux** [comment]
> From @kris7t on gitter: "the cookies in my profile from a previous version of Ferdi work just fine, so it's not a problem with Twitter, only with the login flow itself."
> 
> So there's probably a way to work around the issue by logging in in one browser and then transferring the cookies to Ferdi.

I try to copy twitter cookie from Rambox config folder to Ferdi folder but it doesn't work

**Comment by dbieber** [comment]
It looks like the Cookies are stored in a sqlite3 database named Cookies e.g. in ~/Library/Application Support/Ferdi/Cookies on Mac. Perhaps adding the cookies there could help, but I see there's also a Cookies-journal file, and a SingletonCookie, so it may be more complicated than just inserting the cookie into the database.

**Comment by dbieber** [comment]
Still relevant

**Comment by kris7t** [comment]
This should hopefully be fixed in #1372
